game/theater/iadsnetwork/iadsnetwork.py

```diff
@@ -141,8 +141,9 @@ def update_tgo(self, tgo: TheaterGroundObject) -> None:
                 self.nodes.remove(cn)
         try:
             # Create a new node for the tgo
-            self.node_for_tgo(tgo)
-            # TODO Add the connections or calculate them..
+            node = self.node_for_tgo(tgo)
+            if self.advanced_iads:
+                self._make_advanced_connections(node)
```

Comment:
Only add connections if advanced IADS is active, the method used is the fallback suggested in the open points of #314, i.e. "connect by range"